apps/system/js/notifications.js

```diff
@@ -228,6 +229,7 @@ var NotificationScreen = {
     this._touchStartX = evt.touches[0].pageX;
     this._touchStartY = evt.touches[0].pageY;
     this._touchPosX = 0;
+    this._touchStartDate = Date.now();
```

Comment:
Any reason not to use the event timestamp here? It may well make it safe over clock changes to do so (not sure about that, but using Date.now() definitely isn't)